api/src/opentrons/protocol_api_experimental/pipette_context.py

```diff
@@ -197,8 +197,26 @@ def touch_tip(  # noqa: D102
         v_offset: float = -1.0,
         speed: float = 60.0,
     ) -> PipetteContext:
-        # TODO: https://github.com/Opentrons/opentrons/issues/9526
-        raise NotImplementedError()
+
+        if location is None:
+            raise NotImplementedError("Not including location not yet supported in PE.")
+        if radius != 1.0:
+            raise NotImplementedError("Radius adjustment not yet supported in PE.")
+        if v_offset != -1.0:
+            raise NotImplementedError("Vertical offset not yet supported in PE.")
+        if speed != 60.0:
+            raise NotImplementedError("Speed parameter not yet supported in PE.")
+
+        self._engine_client.touch_tip(
+            pipette_id=self._pipette_id,
+            labware_id=location.parent.labware_id,
+            well_name=location.well_name,
+            well_location=WellLocation(
+                origin=WellOrigin.BOTTOM,
+                offset=WellOffset(x=0, y=0, z=1),
```

Comment:
I believe this should be `-1` rather than `1`, where `-1` means "1 mm closer to the deck" and `1` means "further from the deck". That being said, let's just throw `v_offset` in there since it should just work™️ 

```suggestion
                offset=WellOffset(x=0, y=0, z=v_offset),
```